submissions/kolajoTomike/challenge-array2/question2.js

```diff
@@ -0,0 +1,30 @@
+const getResult = (arr, values) => {
```

Comment:
Nice idea
It fails though for the input `nums = [0,4,3,0], target = 0`. You could look into why that is happening.